README.md

```diff
@@ -20,48 +20,44 @@ The [fal](https://github.com/fal-ai/fal) + [dbt-fal](https://github.com/fal-ai/f
 
 With this combo, you won't have to leave your dbt project and still add more capabilities to your stack.
 
+### Pre-requisites
+
+- [Install Docker](https://docs.docker.com/get-docker/). Check if the installation is ok with `docker --version`.
```

Comment:
@turbo1912 you mentioned one reason why you didn't add Docker before is because you didn't want to require users to install/learn it. That's a fair point, fwiw Docker is completely optional since it just sets up a Postgres database, so I can update the instructions to make that more clear. Thoughts?

Reply:
We can give options:
1. just use the codespace we setup.
2. If you want to run in your environment, use docker

Reply:
I like that, I'll get the Codespace instance working and then I'll update the readme with that as an option.

I think we should also add a callout making it clear that using a local/existing Postgres is also fine.